src/routes/_admin.xpub.tsx

```diff
@@ -0,0 +1,13 @@
+import { createFileRoute } from '@tanstack/react-router';
+
+export const Route = createFileRoute("/_admin/xpub")({
```

Comment:
Why the underscore before `admin` part?
I suppose its good time to re-think the pattern of routes and maybe modify it.